contracts/epochs/FlowClusterQC.cdc

```diff
@@ -258,6 +258,24 @@ access(all) contract FlowClusterQC {
         }
     }
 
+    /// Represents the quorum certificate vote data for a signer 
+    /// of the certificate.
+    access(all) struct ClusterQCVoteData {
```

Comment:
In order to match the [current `util` command output](https://github.com/onflow/flow-go/blob/70f9aabc9ec76bd1a1c23b2bf99880fb2ad4dc2e/cmd/util/cmd/common/clusters.go#L165-L184), I think this structure needs to have the form:
```cadence
struct ClusterQCVoteData {
  voteData: String // hex-encoded aggregated signature
  voterIDs: [String]
}
```